<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogAcquisitionConfiguration</class>
 <widget class="QDialog" name="DialogAcquisitionConfiguration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>682</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>510</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>281</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Acquisition Configuration</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>661</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>Waveforms</string>
   </property>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Record length:</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="spinBoxRecordLenght">
    <property name="geometry">
     <rect>
      <x>116</x>
      <y>34</y>
      <width>71</width>
      <height>29</height>
     </rect>
    </property>
    <property name="minimum">
     <number>64</number>
    </property>
    <property name="maximum">
     <number>4096</number>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>101</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>[max: 1024]</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBoxEnabledChannels">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>661</width>
     <height>351</height>
    </rect>
   </property>
   <property name="title">
    <string>Enabled Channels</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>63</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBoxAFE0">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>120</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>AFE 0</string>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <widget class="QCheckBox" name="checkBoxChannel0">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 0</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel1">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 1</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 2</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 3</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 4</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 5</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 6</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 7</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAFE1">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>30</y>
      <width>120</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>AFE 1</string>
    </property>
    <widget class="QCheckBox" name="checkBoxChannel8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 8</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>90</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 9</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 10</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 11</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 12</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 13</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 14</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 15</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAFE2">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>30</y>
      <width>120</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>AFE 2</string>
    </property>
    <widget class="QCheckBox" name="checkBoxChannel16">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 16</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel17">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 17</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel18">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 18</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel19">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 19</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel20">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 20</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel21">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 21</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel22">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 22</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel23">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 23</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAFE3">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>30</y>
      <width>120</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>AFE 3</string>
    </property>
    <widget class="QCheckBox" name="checkBoxChannel24">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 24</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel25">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 25</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel26">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 26</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel27">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 27</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel28">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 28</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel29">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 29</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel30">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 30</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel31">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 31</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBoxAFE4">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>30</y>
      <width>120</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>AFE 4</string>
    </property>
    <widget class="QCheckBox" name="checkBoxChannel32">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 32</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel33">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 33</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel34">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 34</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel35">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 35</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel36">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 36</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel37">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 37</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel38">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 38</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBoxChannel39">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>101</width>
       <height>26</height>
      </rect>
     </property>
     <property name="text">
      <string>Channel 39</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="pushButtonSelectAll">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>310</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select All</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonUnselect">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>310</y>
      <width>84</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Unselect</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogAcquisitionConfiguration</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogAcquisitionConfiguration</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
